app/javascript/App/startup.js

```diff
@@ -0,0 +1,10 @@
+// react-router app
+import './index'
+
+// non-react-router components
+import './bundles/common/containers/NavUserContainer'
+import './components/CoinShow'
+import './components/IcoFilters'
+import './bundles/common/containers/FlashMessageListContainer'
+import './bundles/common/components/WatchButton'
+import './components/CalendarPage'
```

Comment:
Now, this is the entry point for webpack. If some new separate component will be created it must be imported here in `non-react-router components`.